django/contrib/admin/static/admin/css/vendor/select2/select2.css

```diff
@@ -0,0 +1,482 @@
+.select2-container {
```

Comment:
Although we're vendoring Select2 I don't see a reason to ship the un-minified CSS file with Django.